### DISCUSSION TITLE
Is there a way to set default view for apps in argocd UI?

### DISCUSSION THREAD

**Discussion by codestalkerr** [argoproj/argo-cd#11979] [General]
Today when you login to argocd UI to view an application by default the view is "APPLICATION DETAILS TREE" but I want to have the "APPLICATION DETAILS NETWORK" as default. So is there any way to set that in any config file?

I know that if I select one time it will be saved for next time but all the users needs to do that and I wanted to change the default view as it takes a lot of time to load but with the view we have today and with network view even if we have thousands of apps it loads quickly.

**Comment by nikser** [comment]
Hello

I think you need the article: https://argo-cd.readthedocs.io/en/stable/operator-manual/ui-customization/.

> This can be configured on an Application basis, by setting the `pref.argocd.argoproj.io/default-view annotation`, accepting one of: `tree`, `pods`, `network`, `list` as values.
